src/bentoml_cli/deployment.py

```diff
@@ -42,26 +173,16 @@ def shared_decorator(
         def decorate(f: t.Callable[..., t.Any]) -> t.Callable[..., t.Any]:
             options = [
                 click.argument(
-                    "deployment-name",
+                    "name",
```

Comment:
This is breaking change.

Reply:
Since we are having a pretty big refactor and most things behave differently, I think it's fine to make these breaking changes here and not provide `deployment-name`, `cluster-name`, `kube-namespace` anymore?

Reply:
ok sg.